ihttp/ihttp.go

```diff
@@ -96,13 +97,20 @@ func (s Handler) Handle(w http.ResponseWriter, req *http.Request) {
 		http.NotFound(w, req)
 		return
 	}
-	b, err := w.Write(file)
-	if err != nil {
-		log.Error(err, "error serving file")
-		w.WriteHeader(http.StatusInternalServerError)
-		return
+	w.Header().Set("Content-Length", fmt.Sprintf("%d", len(file)))
+	switch req.Method {
+	case http.MethodHead:
+		w.WriteHeader(http.StatusOK)
+		log.Info("HEAD method requested", "fileSize", len(file))
+	default:
+		b, err := w.Write(file)
+		if err != nil {
+			log.Error(err, "error serving file")
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+		log.Info("file served", "bytesSent", b, "fileSize", len(file))
 	}
```

Comment:
I see a comment that says handle http.MethodGet and http.MethodHead.  But don't see for case "http.MethodGet" ?

Reply:
the get method was covered by the default case. I've updated this though to be more explicit. what do you think?

Reply:
Do you think want to add log in default?